fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ParameterVisitorBatchDAO.java

```diff
@@ -439,7 +439,7 @@ public void visit(TokenParmVal param) throws FHIRPersistenceException {
             // Add the new token value to the collection we're building...what's the resourceTypeId?
             final int resourceTypeId = identityCache.getResourceTypeId(param.getResourceType());
             if (tokenValue == null) {
-                logger.info("tokenValue is NULL for: " + parameterName + "[" + parameterNameId + "], " + codeSystem);
```

Comment:
should we add isLoggable to the if or use your `() ->` trick?

Reply:
updated per latest commit